frontend/src/app/dataset/dataset.component.ts

```diff
@@ -60,11 +61,29 @@ export class DatasetComponent {
     this.plotRef = ref.first;
   }
 
+  /**
+   * Setter method to initialize the dataset with a reference to itself. This enables
```

Comment:
`Initializes the dataset...` instead of `Setter method to initialize the dataset`